### PR TITLE
Lecture 08-a

### PR DESCRIPTION
**Pull request by DikshaMeghwal**
In this lecture we start with a recap of the energy based models, their underlying principles. We then discuss the concept of latent variables often involved while designing energy based models. We look at several examples for energy based models like K-means clustering, speech recognition, keyword recognition, semantic segmentation. We talk about the technique of marginalization for drawing inference for probabilistic based models. We then discuss how to train energy based models and compare several choice of loss functions.

-- First part of Lecture 8 (01 Apr 2019) by Imran Imran and Chiao-Hsun Wang Edited by Diksha Meghwal

### CODE REVIEW

**Comment by Atcold** [comment]
PR summary, author list, and date are missing.
Plz, edit the first message, similarly to how it's done in #1.

**Comment by bellineq** [comment]
In this lecture note we discuss energy based models, several applications of energy based models,
and how to train an energy based model, specifically on comparing choice of loss function.

-- First part of Lecture 8 (01 Apr 2019) by  Imran Imran and Chiao-Hsun Wang Edited by Diksha Meghwal

**Comment by Atcold** [comment]
The summary should be around 50/70 words.

**Comment by DikshaMeghwal** [comment]
In this lecture we start with a recap of the energy based models,  their underlying principles. We then discuss the concept of latent variables often involved while designing energy based models. We look at several examples for energy based models like K-means clustering, speech recognition, keyword recognition, semantic segmentation. We talk about the technique of marginalization for drawing inference for probabilistic based models. We then discuss how to train energy based models and compare several choice of loss functions.
-- First part of Lecture 8 (01 Apr 2019) by Imran Imran and Chiao-Hsun Wang Edited by Diksha Meghwal

File: lectures/08-a/theory.tex [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+% Lecture date: 04/01/2019
+
+A quick recap, energy based models(EBMs) are functions that take an input and product a scalar output as a result of some optimization. Eg:
+
+\begin{figure}[htb]
+    \centering
+    \begin{subfigure}[b]{0.25\textwidth}
+        \includegraphics[width=\textwidth]{lectures/08-a/images/conditional_EBM.png}
+        \caption{Conditional model}
+        \label{fig:conditional}
```
Comment (Atcold) [inline lectures/08-a/theory.tex:16]: This is already defined in `lectures/07-b`. Change it.

Comment (bellineq) [reply lectures/08-a/theory.tex]: Hi Alfred,
It seems like you've applied the change,
do we have to add anything else?
Thanks!

**Pull request merged**